mlflow/entities/trace.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
+
+import json
+from dataclasses import asdict, dataclass
+
+from mlflow.entities._mlflow_object import _MLflowObject
+from mlflow.entities.trace_data import TraceData
+from mlflow.entities.trace_info import TraceInfo
+
+
+@dataclass
+class Trace(_MLflowObject):
```

Comment:
could you add the `_repr_mimebundle_()` function here:

https://github.com/mlflow/mlflow/blob/tracing/mlflow/tracing/types/model.py#L25-L38

Reply:
Ah great catch, will add!